printer/downloadFile.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+const fs = require('fs');
+const { v4: uuidv4 } = require('uuid');
+const logger = require('../util/logger.js');
+/**
+ * Download PDF file from URL.
+ * @param {Sring} url : PDF file url
+ * @returns Promise return the path to downloaded PDF file
+ * if it was successfully download. It then returns false if there
+ * was an error.
+ */
+async function downloadFileFromURL(url) {
+  return new Promise( ( resolve ) => {
+    axios({
+      url: url,
+      method: 'GET',
+      responseType: 'stream'
+    })
+      .then(response => {
+        const fileName = uuidv4();
+        const filePath = `/tmp/${fileName}.pdf`;
+        fs.writeFileSync(filePath, '');
+        const fileStream = fs.createWriteStream(filePath);
+        response.data.pipe(fileStream);
+        logger.info('Downloaded file location: ', filePath);
+        resolve(filePath);
+      })
+      .catch(error => {
+        logger.error('Download Unsuccessfully! ', error);
+        resolve(false);
+      });
+  });
+}
+
+module.exports = downloadFileFromURL;
```

Comment:
if u wanna import it like
```js
const { downloadFileFromURL } = require('./downloadFile');
```

then we must do
```suggestion
module.exports = { downloadFileFromURL };
```